citar-file.el

```diff
@@ -229,6 +233,25 @@ need to scan the contents of DIRS in this case."
                  (puthash key (nreverse filelist) files))
                files))))
 
+(defun citar-file--has-file-notes-hash ()
+  "Return a hash of keys and file paths for notes."
+  (citar-file--directory-files
+   citar-notes-paths nil citar-file-note-extensions))
```

Comment:
Any reason not to include `citar-file-additional-files-separator` as an argument here?

Reply:
I think I had mistakenly assumed that was only for library files?

Reply:
Fixed.